lib/klife/connection/supervisor.ex

```diff
@@ -12,7 +12,8 @@ defmodule Klife.Connection.Supervisor do
   def init(opts) do
     children = [
       {Klife.Connection.BrokerSupervisor, opts},
-      {Klife.Connection.Controller, opts}
+      {Klife.Connection.Controller, opts},
+      {Task.Supervisor, name: Klife.TaskSupervisor}
```

Comment:
If we gonna use this supervisor only to handle broker message callbacks we should name it more specifically. Something like `Klife.Connection.CallbackSupervisor` would be better IMO.

Anyway, we can use a more general task supervisor if we want, but them we need to initialize it as a child of the `Klife.Application` instead of `Klife.Connection.Supervisor`.